src/main/java/com/genesys/workspace/TargetsApi.java

```diff
@@ -43,10 +51,123 @@ public TargetSearchResult search(String searchTerm, TargetsSearchOptions options
                     targets.add(new Target(t));
                 }
             }
-            return new TargetSearchResult(data.getTotalMatches(), targets);
+            return new SearchResult(data.getTotalMatches(), targets);
 
         } catch (ApiException e) {
             throw new WorkspaceApiException("searchTargets failed.", e);
         }
     }
+
+    public void addRecentTarget(TargetInformation target, RecentData recentData) throws WorkspaceApiException {
```

Comment:
Can we accept (Target target, String media, BigDecimal timestamp) for this and create TargetInformation and RecentData inside? Maybe we could also have an override that doesn't require timestamp and we grab current time.

For recently used items, the API user is going to have a list of target search results already so it would be nice if they could just pass back the same one they used rather than translating to TargetInformation.